### PR TITLE
Project 2: Processing data in models.

### PR DESCRIPTION
**Pull request by dfsalam**
# In this PR I:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,108 @@
...
+- **Add comments**
 
-* Deployment instructions
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* ...
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Video presentation](https://drive.google.com/file/d/1fpKi5678kdljgkF2E-YdDqBEuovNRckl/view?usp=sharing)
+
```
Comment (mahbubul14) [inline README.md]: - [ ] Your project has no live demo link yet. You can remove the link for now. :+1:

File: README.md [hunk 2]
```diff
@@ -1,24 +1,108 @@
...
+### Setup
+
+Clone this repository to your desired folder:
+
+- git clone https://github.com/dfsalam/catalog-of-my-things.git
+- cd catalog-of-my-things
+- code . <!-- If you are using VS Code -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (mahbubul14) [inline README.md]: - [ ] Kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @dfsalam,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Well-implemented processing data in models.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (dfsalam) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (dfsalam) [reply README.md]: Thanks for your review. Done.

**Review by prolajumokeoni — APPROVED** [review]
Hi @dfsalam 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 👏🏿  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**